backend/Squirrel.ConsoleApp/Services/PostgreSqlService.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Npgsql;
+using Squirrel.ConsoleApp.Exceptions;
+using Squirrel.ConsoleApp.Interfaces;
+using System.Text;
+
+namespace Squirrel.ConsoleApp.Services
+{
+    public class PostgreSqlService : IDatabaseService
+    {
+        private string _connectionString;
+
+        public string ConnectionString
+        {
+            get { return _connectionString; }
+            set { _connectionString = value; }
+        }
+
+        public PostgreSqlService(string connectionString)
+        {
+            _connectionString = connectionString;
+        }
+
+        public async Task<string> ExecuteQueryAsync(string query)
```

Comment:
Is it possible to return non string results?

Reply:
I mean here, that you convert a table of results to a string, why not to pass some parse function or make it generic in some way?